scripts/test_runner.sh

```diff
@@ -82,21 +82,21 @@ if [ -n "${TEST_DIR}" ]; then
             [ -n "${RANCHERVERSION}" ] && OPTS+=(-rancherVersion "${RANCHERVERSION}")
       go test "${OPTS[@]}"
     elif [ "${TEST_DIR}" = "dualstack" ]; then
-        go test -timeout=65m -v -count=1 ./entrypoint/dualstack/...
+        go test -timeout=65m -v -count=1 ./entrypoint/dualstack/... -destroy "${DESTROY}"
```

Comment:
if you are adding ` -destroy "${DESTROY}"` here on test runner, you need to also refactor makefile + docker run file which are the consumers of it 

right now you will get an error trying to start tests locally cause is expecting this flag

Reply:
I though we need to add it here for the batch test to work but realize now that its set to true in the suite itself, so I guess I can remove it then from here

Reply:
jenkins dont use this file